### PR TITLE
[enhancement] add more content of gitignore file

### PR DESCRIPTION
**Pull request by jackwener**
## Proposed changes

Ignore the `target` file in samples/doris-demo/

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)
- [ ] Optimization. Including functional usability improvements and performance improvements.
- [ ] Dependency. Such as changes related to third-party components.
- [ ] Other.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

## Further comments

None

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -65,7 +65,13 @@ docs/package-lock.json
 fe/fe-common/.classpath
 rpc_data/
 
+## ignore sample target
```
Comment (yangzhg) [inline .gitignore:68]: you crash those to samples/doris-demo/*/target

File: .gitignore [hunk 2]
```diff
@@ -65,7 +65,13 @@ docs/package-lock.json
...
+samples/doris-demo/spring-jdbc-demo/target
+samples/doris-demo/flink-demo/target
+samples/doris-demo/spark-demo/target
 
-#ignore vscode project file
+# ignore vscode project file
 .vscode
-samples/doris-demo/stream-load-demo/target
+
+
```
Comment (yangzhg) [inline .gitignore:77]: Just one blank line is enough

Comment (jackwener) [reply .gitignore:74]: Fixed.

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**